code/game/area/Space Station 13 areas.dm

```diff
@@ -294,6 +294,17 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 /area/shuttle/civvie/cockpit
 	name = "\improper Civilian Transport Cockpit"
 
+///Centuries old Civilian Shuttle
```

Comment:
I dislike the naming, perhaps something like "Century shuttle" would be better (century old suggests that its very outdated)

Reply:
Changed the name,

![image](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/assets/121488347/31a237f1-c186-45da-9475-4a2ad7b4d28e)